tests/builder/test_base.py

```diff
@@ -202,7 +202,8 @@ def test_replace_macros(macro_spec_path, prepared_macros):
     result_macros = dict(MACRO_REGEX.findall(spec_contents))
 
     # All expected macros have their contents replaced
-    assert all(val == 'REPLACED' for val in result_macros.values()), result_macros
```

Comment:
The removed line was the line that you added on another commit on this PR.
I do not think this commit is meaningful for users that will see your commit from outside.
Please squash your commits with proper units. Please check other commit units in this PR.